content/articles/getting-started-with-cypress.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Getting Started With Cypress"
+date: 2019-11-07
+draft: true
+type: "post"
+authors: ["buonincontri"]
+tags: [""]
+lede: "Trying out Cypress for Integration testing"
+shortlede: "Testing with Cypress"
+poster: "poster-get-started-with-cypress.jpg"
+socmediaimg: ""
+hiliteimg: "poster-get-started-with-cypress.jpg"
+poster_sourceurl: "https://commons.wikimedia.org/wiki/File:Wheat-Field-with-Cypresses-(1889)-Vincent-van-Gogh-Met.jpg"
+poster_source: "Wikimedia"
+bookendanimal: "cat"
+---
+Testing is a key pillar of writing reliable software. When we build Django apps
+at the CTL, the testing story is pretty clear - its focused on unit testing.
+But what about other kinds of sites, like Hugo sites or React apps? Work on a
+[recent project](https://ohcoe.ctl.columbia.edu) led me to try the Cypress
+testing framework, and I was happy that I did.
+
+## Why Cypress?
+Cypress is essentially the Chrome browser married with a Javascript testing
```

Comment:
"essentially" -- why do you need that word?

Reply:
fluff - it's gone